### ISSUE TITLE
Create "Heads-up display" type UI

### ISSUE THREAD

**Issue by groovecoder** [groovecoder/fx-doiuse#13]
Still need to think of the best way to do this, but probably a sticky header or footer with details on clicking.

**Comment by groovecoder** [comment]
@openjck - I find myself fumbling clumsily with this task. If you have any time, would you be interested in this kind of front-end work for the add-on? Could be laying the ground-work for future HUD UI too ...

**Comment by openjck** [comment]
Yeah, I would love to be part of the conversation about UI. Can you say more about the heads-up concept? It sounds interesting. What advantages would it have over the usual Developer Tools approach of showing similar types of reports (console logs, performance, network activity, etc.) in the toolbar?

We can also think about prototyping an early heads-up display (maybe a sticky header or footer like you mention) and iterating from there.
